front-end/components/Auth/LoginView.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import loginIcon from './static/LoginIcon.png';
+import TextEntry from '../shared/TextEntry.jsx';
+import './static/AuthStyles.css'
+// Convert to absolute import later
+
+/**
+ * @param:
+ * @returns: [html div] which spans all allotted space
+ */
+
+class LoginView extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.loginUsername = '';
+    this.loginPassword = '';
+  }
+
+  setUsername = (value) => {
+    this.loginUsername = value;
+    // this.setState({ loginUsername: value });
```

Comment:
For some reason using setState changes the input incorrectly. I'll show you if curious

Reply:
Also any idea why you can only access state through arrow functions? otherwise you get undefined.

Reply:
React invokes whatever function is passed in as a property.  Therefore the function needs to be an anom/arrow function.  It cannot invoke a declaration such as `this.someProperty = 12` because this is not a function. Alternatively,  `() => { this.state.someProperty = 12 }` is an invokable closure which does the same thing.  One reason they may have chosen to proceed this way is because declarations can easily become anom/arrow functions, but it can be more difficult to turn an arrow function into a declaration (because it could require evaluation).